docs/how-to-setup-wsl.md

```diff
@@ -126,6 +128,65 @@ Now that you have installed the pre-requisites, follow [our local setup guide](h
 >
 > Please note, at this time the setup for Cypress tests (and related GUI needs) is a work in progress. You should still be able to work on most of the codebase.
 
+## Optimize Windows and WSL
+
+   > [!NOTE]
+   >
+   > The following tips were collected from across the web and have not gone through vigorous testing. Your mileage may vary.
+
+### Adjust processer scheduling for background services
+
+This may reduce incidents of Docker containers crashing due to lack of resources.
+
+Open the System Properties control panel by pressing <kbd>Win + R</kbd> and entering `sysdm.cpl`
+
+![Run dialog](images/wsl/run-sysdm.png)
+
+Go to Advanced -> Performance -> Settings…
+
+![Performance settings under Advanced tab](images/wsl/advanced-performance-settings.png)
+
+Under Advanced -> Processor scheduling, choose "Background services". Do not close the window. Continue to the next tip.
+
+![Background services](images/wsl/background-services.png)
+
+### Increase the size of Windows paging file for the system drive
+
+Under Advanced -> Virtual memory, click "Change…"
+
+![Change virtual memory under Advanced tab](images/wsl/advanced-virtual-memory.png)
+
+Choose "Custom size". Set the initial size to 1.5x and the maximum size to 3x of your physical memory. Then click "Set".
+
+![Set custom size](images/wsl/set-custom-size.png)
+
+### Increase the size of memory allocated to WSL
+
+Create a [`.wslconfig` file](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#configuration-setting-for-wslconfig) in your [`%UserProfile%` directory](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#wslconfig) (typically `C:\Users\<UserName>\.wslconfig`) with the following content (assuming 16GB of physical memory):
+
+```ini
+# Settings apply across all Linux distros running on WSL 2
+[wsl2]
+
+# Limits VM memory to use no more than 12 GB, this can be set as whole numbers using GB or MB
+memory=12GB
+
+# Sets amount of swap storage space to 8GB, default is 25% of available RAM
+swap=8GB
+```
```

Comment:
For this, I suggest just mentioning default memory might be an issue and pointing to the docs: https://learn.microsoft.com/en-us/windows/wsl/wsl-config#configuration-setting-for-wslconfig

There are some gotchas the docs mention that Campers should be aware of before they blindly copy `memory=12GB` in their own config.